### ISSUE TITLE
[Bug]: librosa.griffinlim_cqt produces AttributeError: 'csr_matrix' object has no attribute 'H'

### ISSUE THREAD

**Issue by malikzh** [librosa/librosa#1871]
### Bug summary

Code from the example about griffinlim_cqt produces error
I took the example from this place: https://librosa.org/doc/main/generated/librosa.griffinlim_cqt.html

### Code for reproduction

```Python
import librosa
import numpy as np

y, sr = librosa.load(librosa.ex('trumpet', hq=True), sr=None)
# Get the CQT magnitude, 7 octaves at 36 bins per octave
C = np.abs(librosa.cqt(y=y, sr=sr, bins_per_octave=36, n_bins=7*36))
# Invert using Griffin-Lim
y_inv = librosa.griffinlim_cqt(C, sr=sr, bins_per_octave=36)
```

### Actual outcome

```
Traceback (most recent call last):
  File "/Users/malikzh/PycharmProjects/torchguitar/test.py", line 8, in <module>
    y_inv = librosa.griffinlim_cqt(C, sr=sr, bins_per_octave=36)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/malikzh/PycharmProjects/torchguitar/.venv/lib/python3.11/site-packages/librosa/core/constantq.py", line 1423, in griffinlim_cqt
    inverse = icqt(
              ^^^^^
  File "/Users/malikzh/PycharmProjects/torchguitar/.venv/lib/python3.11/site-packages/librosa/core/constantq.py", line 711, in icqt
    inv_basis = fft_basis.H.todense()
                ^^^^^^^^^^^
AttributeError: 'csr_matrix' object has no attribute 'H'
```

### Expected outcome

Expected no error

### Additional information

I tried with different versions of the Python interpreter (3.11.7 and 3.12.0) and I tried run this code snippet from the Mac OS and Windows. In both OS I used specified above versions.

### Operating system

Mac OS, Windows 10 (through WSL2 Ubuntu)

### software versions

```
macOS-14.2.1-arm64-arm-64bit
Python 3.11.7 (v3.11.7:fa7a6f2303, Dec  4 2023, 15:22:56) [Clang 13.0.0 (clang-1300.0.29.30)]
NumPy <module 'numpy.version' from '/Users/malikzh/PycharmProjects/torchguitar/.venv/lib/python3.11/site-packages/numpy/version.py'>
SciPy <module 'scipy.version' from '/Users/malikzh/PycharmProjects/torchguitar/.venv/lib/python3.11/site-packages/scipy/version.py'>
librosa <module 'librosa.version' from '/Users/malikzh/PycharmProjects/torchguitar/.venv/lib/python3.11/site-packages/librosa/version.py'>
INSTALLED VERSIONS
------------------
python: 3.11.7 (v3.11.7:fa7a6f2303, Dec  4 2023, 15:22:56) [Clang 13.0.0 (clang-1300.0.29.30)]

librosa: 0.10.2.post1

audioread: 3.0.1
numpy: 2.0.2
scipy: 1.14.1
sklearn: 1.5.1
joblib: 1.4.2
decorator: 5.1.1
numba: 0.60.0
soundfile: 0.12.1
pooch: v1.8.2
soxr: 0.5.0.post1
typing_extensions: installed, no version number available
lazy_loader: 0.4
msgpack: 1.0.8

numpydoc: None
sphinx: None
sphinx_rtd_theme: None
matplotlib: None
sphinx_multiversion: None
sphinx_gallery: None
mir_eval: None
ipython: None
sphinxcontrib.rsvgconverter: None
pytest: None
pytest_mpl: None
pytest_cov: None
samplerate: None
resampy: None
presets: None
packaging: 24.1
```

### Installation

pip

**Comment by malikzh** [comment]
## On Windows host (through WSL2 Ubuntu)

### Code

```python
import librosa
import numpy as np

y, sr = librosa.load(librosa.ex('trumpet', hq=True), sr=None)
# Get the CQT magnitude, 7 octaves at 36 bins per octave
C = np.abs(librosa.cqt(y=y, sr=sr, bins_per_octave=36, n_bins=7*36))
# Invert using Griffin-Lim
y_inv = librosa.griffinlim_cqt(C, sr=sr, bins_per_octave=36)
```

### Actual outcome

```
malikzh@MALIK-PC:~/projects/melody$ /bin/python /home/malikzh/projects/melody/test.py
Traceback (most recent call last):
  File "/home/malikzh/projects/melody/test.py", line 8, in <module>
    y_inv = librosa.griffinlim_cqt(C, sr=sr, bins_per_octave=36)
  File "/home/malikzh/.local/lib/python3.10/site-packages/librosa/core/constantq.py", line 1423, in griffinlim_cqt
    inverse = icqt(
  File "/home/malikzh/.local/lib/python3.10/site-packages/librosa/core/constantq.py", line 711, in icqt
    inv_basis = fft_basis.H.todense()
AttributeError: 'csr_matrix' object has no attribute 'H'
```

### Operating system

```
Microsoft Windows [Version 10.0.19045.4780]

WSL2 OS: 
malikzh@MALIK-PC:~/projects/melody$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.3 LTS
Release:        22.04
Codename:       jammy
```

### Software version on WSL

```
Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.35
Python 3.10.12 (main, Mar 22 2024, 16:50:05) [GCC 11.4.0]
NumPy <module 'numpy.version' from '/home/malikzh/.local/lib/python3.10/site-packages/numpy/version.py'>
SciPy <module 'scipy.version' from '/home/malikzh/.local/lib/python3.10/site-packages/scipy/version.py'>
librosa <module 'librosa.version' from '/home/malikzh/.local/lib/python3.10/site-packages/librosa/version.py'>
INSTALLED VERSIONS
------------------
python: 3.10.12 (main, Mar 22 2024, 16:50:05) [GCC 11.4.0]

librosa: 0.10.2.post1

audioread: 3.0.1
numpy: 1.26.4
scipy: 1.14.0
sklearn: 1.5.1
joblib: 1.4.2
decorator: 5.1.1
numba: 0.60.0
soundfile: 0.12.1
pooch: v1.8.2
soxr: 0.3.7
typing_extensions: installed, no version number available
lazy_loader: 0.4
msgpack: 1.0.8

numpydoc: None
sphinx: None
sphinx_rtd_theme: None
matplotlib: 3.9.1
sphinx_multiversion: None
sphinx_gallery: None
mir_eval: None
ipython: None
sphinxcontrib.rsvgconverter: None
pytest: None
pytest_mpl: None
pytest_cov: None
samplerate: None
resampy: None
presets: None
packaging: 24.1
```

**Comment by bmcfee** [comment]
Redundant to #1849 ; fixed by #1851
